numpy/core/_methods.py

```diff
@@ -189,10 +204,14 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False):
     # Compute the mean.
     # Note that if dtype is not of inexact type then arraymean will
     # not be either.
-    arrmean = umr_sum(arr, axis, dtype, keepdims=True)
+    arrmean = umr_sum(arr, axis, dtype, keepdims=True, where=where)
+    # The shape of rcount has to match arrmean to not change the shape of out
+    # in broadcasting. Otherwise, it cannot be stored back to arrmean.
+    div = (rcount if isinstance(rcount, nt.intp)
```

Comment:
```suggestion
    div = (rcount if rcount.ndim == 0
```

Reply:
:+1: nice, then it also fits on one line.